src/Counter/CounterForm.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import ButtonList from "./ButtonList";
+
+class CounterForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      counter: 0
+    };
+  }
+  handler = (val) => {
+    this.setState({ counter: (this.state.counter + val) })
+  }
+
+  render() {
+    return <div>
+      <h2>{this.state.counter}</h2>
+      <ButtonList onClickHandler={this.handler} />
```

Comment:
according to the task we need to create elements using createElement API 
<img width="977" alt="image" src="https://github.com/GaKhalatyan86/ReactLessons/assets/36797396/1d445fc3-8c0f-4a5c-a867-7ff01c1e5fa0">

This doc will help you 
https://react.dev/reference/react/createElement

Reply:
fixed - updated with createElement